tests/integration/cgroups.bats

```diff
@@ -21,15 +21,10 @@ function setup() {
     set_cgroups_path "$BUSYBOX_BUNDLE"
 
     # Set some initial known values
-    DATA=$(cat <<-EOF
-    "memory": {
-        "kernel": 16777216,
-        "kernelTCP": 11534336
-    },
-EOF
-    )
-    DATA=$(echo ${DATA} | sed 's/\n/\\n/g')
-    sed -i "s/\(\"resources\": {\)/\1\n${DATA}/" ${BUSYBOX_BUNDLE}/config.json
+    cat "${BUSYBOX_BUNDLE}/config.json" \
+	| jq '.linux.resources.memory |= {"kernel": 16777216, "kernelTCP": 11534336}' \
+	> "${BUSYBOX_BUNDLE}/config.json.tmp"
+    mv "${BUSYBOX_BUNDLE}/config.json"{.tmp,}
```

Comment:
if we're going to make this a pattern, can we use `sponge` so you don't have to have this `mv` step each time? `sponge` is like `tee` except it waits until EOF to overwrite the file (thus allowing you to replace files that are used as input to the pipeline).

Reply:
It also kind of feels like `cat | jq | sponge foo` should be a function or something.

Reply:
Ok no problem! I'll look into it.

Reply:
There is a relevant discussion in here. Also, `sponge` can be implemented as a one-liner awk (in case it's not readily available on some distro we test on): https://github.com/stedolan/jq/issues/105

Reply:
Also, `cat` is not needed, jq is capable of reading from a file.

Reply:
+1 for sponge (`moreutils`), it seems universally available on all distros

Reply:
sponge doesn't seem to work with the CI tests. I'm going to use the one-liner awk.